contrib/opentimelineio_contrib/application_plugins/rv/example_otio_reader/otio_reader.py

```diff
@@ -250,19 +262,28 @@ def _create_item(it, track_kind=None):
     if hasattr(it, "media_reference") and it.media_reference:
         _add_metadata_to_node(it.media_reference, src)
 
-    # because OTIO has no global concept of FPS, the rate of the duration is
-    # used as the rate for the range of the source.
-    # RationalTime.value_rescaled_to returns the time value of the object in
-    # time rate of the argument.
-    cut_in = range_to_read.start_time.value_rescaled_to(
-        range_to_read.duration
-    )
-    commands.setIntProperty(src + ".cut.in", [int(cut_in)])
+    in_frame = out_frame = None
+    if hasattr(it, "media_reference") and it.media_reference:
+        if isinstance(it.media_reference, otio.schema.ImageSequenceReference):
+            in_frame, out_frame = \
+                it.media_reference.frame_range_for_time_range(
+                    range_to_read
+                )
+
+    if not in_frame and not out_frame:
+        # because OTIO has no global concept of FPS, the rate of the duration
+        # is used as the rate for the range of the source.
+        # RationalTime.value_rescaled_to returns the time value of the object
+        # in time rate of the argument.
+        in_frame = range_to_read.start_time.value_rescaled_to(
+            range_to_read.duration
```

Comment:
I think you may want to use `range_to_read.duration.rate` here (assuming `range_to_read` is a `TimeRange`). As is, this will cause an implicit conversion of the duration rational to float and use that float as the rate.

Reply:
Sure, however this was simply a copy of the original external rv adapter:

https://github.com/PixarAnimationStudios/OpenTimelineIO/blob/imagesequence/contrib/opentimelineio_contrib/adapters/extern_rv.py#L316-L320

We should just keep them in sync.

Reply:
Which I again kept from the original adapter pre ImageSequenceReference. 🙄

Reply:
Hrm, okay. I'm a little confused now. Reading in more detail, I'd expect this code to look a little more like:
```
in_frame = otio.opentime.to_frames(range_to_read.start_time, fps=range_to_read.duration.rate)
out_frame = otio.opentime.to_frames(range_to_read.end_time_inclusive(), fps=range_to_read.duration)
```

Reply:
We figured out why this code looks funny. `value_rescaled_to` is overloaded to accept either a rate, or a RationalTime, in which case it will use the rate from that RationalTime. So passing `duration` does the same thing as passing `duration.rate`.

Using `to_frames` would be fine too if you want to ensure that you snap to an integer frame number.

Reply:
Something like this? fdcda6771fdaaf870a511e72243a8032cc754f80

Reply:
I like it! I think it clarifies the intent of the code.

Reply:
@jminor @reinecke Should I do this in the adapter as well?